### PR TITLE
optionally show embeds for /pep

### LINKED ISSUE #40: Embed reply for PEP detection
- [ ]  Add an option to allow the default link embed in the `/pep` command
- [ ] During detection for a message with pep in it, instead of sending links, send one embed listing all PEPs that were mentioned and their respective links.

> nah, but I was thinking about making the entire reply an embed so you can have hyperlinks when multiple PEPs are mentioned maybe

_Originally posted by @trag1c in https://github.com/mcb-dev/mCodingBot/pull/39#discussion_r968876667_

### PR DESCRIPTION
**Pull request by CircuitSacul**
closes #40

### CODE REVIEW

File: mcodingbot/plugins/python.py [hunk 1]
```diff
@@ -10,8 +10,9 @@
 PEP_REGEX = re.compile(r"pep *(?P<pep>\d{1,4})", re.IGNORECASE)
 
 
-def get_pep_link(pep_number: int) -> str:
-    return f"<https://peps.python.org/pep-{pep_number:04}/>"
+def get_pep_link(pep_number: int, hide_embed: bool) -> str:
```
Comment (Lunarmagpie) [inline mcodingbot/plugins/python.py:13]: I prefer this signature for functions like this
```suggestion
def get_pep_link(pep_number: int, *, hide_embed: bool) -> str:
```

**Review by Lunarmagpie — CHANGES_REQUESTED** [review]

File: mcodingbot/plugins/python.py [hunk 2]
```diff
@@ -22,9 +23,14 @@ class PEPCommand:
     pep_number = crescent.option(
         int, "The PEP number.", name="pep-number", min_value=0, max_value=9999
     )
+    hide_embed = crescent.option(
```
Comment (qexat) [inline mcodingbot/plugins/python.py:26]: Imo, it should be the opposite (aka `show_embed` defaulting to `True`) but that's up to code style

Comment (CircuitSacul) [reply mcodingbot/plugins/python.py:26]: For the command option, I agree. I'll change that.

**Review by Lunarmagpie — APPROVED** [review]

File: mcodingbot/plugins/python.py (hunk 1 above)
Comment (qexat) [reply mcodingbot/plugins/python.py]: boolean args should most of the time be keyword-only yeah

**Review by Endercheif — APPROVED** [review]

File: mcodingbot/plugins/python.py [hunk 3]
```diff
@@ -42,7 +48,7 @@ async def on_message(event: hikari.MessageCreateEvent) -> None:
         return
 
     pep_links_message = "\n".join(
-        f"PEP {pep_number}: {get_pep_link(pep_number)}"
+        f"PEP {pep_number}: {get_pep_link(pep_number, hide_embed=True)}"
```
Comment (qexat) [inline mcodingbot/plugins/python.py:51]: do you mean, `hide_embed=False`?

Comment (qexat) [reply mcodingbot/plugins/python.py:51]: honestly it would be better to just call it `show_embed` as well

Comment (CircuitSacul) [reply mcodingbot/plugins/python.py:51]: No I didn't mean that, the reply can already be annoying, even without the embed.

**Pull request merged**